CMakeLists.txt

```
@@ -76,7 +76,7 @@ if(NOT MSVC)
     list(INSERT flags_to_test 0 -Wall)
         option(NATIVE "Build for native performance (march=native)" ON)
```

```suggestion
        option(NATIVE "Build for native performance (march=native)")
```
It would instead be better to do this to not build for the native cpu arch by default